build-tools/packages/build-cli/src/lib/git.ts

```diff
@@ -187,16 +187,32 @@ export class Repository {
 	 * @returns An array of all commits between the base and head commits.
 	 */
 	public async revList(baseCommit: string, headCommit: string = "HEAD"): Promise<string[]> {
-		const result = await this.git.raw("rev-list", `${baseCommit}..${headCommit}`);
+		const result = await this.git.raw("rev-list", `${baseCommit}..${headCommit}`, "--reverse");
```

Comment:
Is this called anywhere else? Since the order is being changed, make sure no other callers are going to break because they assume the other order.

Reply:
revList is not called anywhere else